packages/fixtures/src/news-and-events.ts

```diff
@@ -1,10 +1,11 @@
-import { NewsOrEventResponse } from '@asap-hub/model';
+import { NewsAndEventsType, NewsOrEventResponse } from '@asap-hub/model';
 
 export const createNewsAndEventsResponse = (
   key: string,
+  responseType: NewsAndEventsType = 'News',
 ): NewsOrEventResponse => ({
   id: `uuid-${key}`,
```

Comment:
Might be worth just adding the `responseType` into the key so we don't get duplicate Ids which i can see is handled above but just for someone in the future.